cime_config/cesm/machines/config_batch.xml

```diff
@@ -91,8 +91,8 @@
       <directive                       > -R "span[ptile={{ ptile }}]"</directive>
       <directive                       > -N  </directive>
       <directive default="poe"         > -a {{ poe }} </directive>
-      <directive default="cesm.stdout" > -o {{ cesm_stdout }}.%J  </directive>
-      <directive default="cesm.stderr" > -e {{ cesm_stderr }}.%J  </directive>
+      <directive default="cesm.stdout" > -o {{ output_error_path }}.%J  </directive>
```

Comment:
typically we put them in the same file - not sure why they were separated just for yellowstone, don't need to be,